rfc/rfc022-discovery-service.md

```diff
@@ -48,16 +48,21 @@ All presentations MUST conform to the Presentation Definition associated with th
 
 Presentations MUST be encoded in JWT format as string ([JWT]).
 
+The protocol identifies the server and client roles. A client can be a server and vice versa. The individual operations define what to do in certain cases.
+
 ### 3.1 Registration
 
-Clients can publish a Verifiable Presentation to the list by sending it in an HTTP POST to the service endpoint.
+Clients can publish a Verifiable Presentation to the server by sending it in an HTTP POST to the service endpoint.
 The HTTP request body MUST be a Verifiable Presentation (in JWT format). The content type of the request MUST be `application/json`.
 
 The server MUST validate the Verifiable Presentation as specified in section 4. If the validation fails, it MUST return a 400 Bad Request response.
-If the validation succeeds, the Verifiable Presentation MUST be added to the list and the server MUST return a 201 Created response.
+If the validation succeeds, the Verifiable Presentation MUST be added to the internal list and the server MUST return a 201 Created response.
+If the Verifiable Presentation is already on the list, the server MUST return a 304 Not Modified response.
+If the server is configured as client, it MUST send the presentation to the server it has configured.
```

Comment:
"server is configured as client"?

A node is either client, or client and server (because client functionality is always enabled when a Service Definition is loaded)